README.md

```diff
@@ -61,6 +63,7 @@ The process of setting up is largely the same for different operating systems. H
 Go to  https://visualstudio.microsoft.com/visual-cpp-build-tools/ and click 'Download Build Tools'.
 Open the installer and make sure you are installing the package labeled *C++ build tools*, then follow the prompts.
     - **(If installing on CAC, you will need the admin password and must contact support@ionpath.com)**
+    - **Git - CAC:** We highly recommend installing git system-wide on the CAC, by downloading the installation utility [here](https://git-scm.com/downloads).
```

Comment:
More handholding of what they need to do to install it, screenshots, which link, etc etc